admin/Pipfile

```diff
@@ -23,6 +23,7 @@ notifications-python-client = "==4.8.1"
 notifications-utils = {path = "./../utils",editable = true}
 sentry-sdk = {extras = ["flask"]}
 itsdangerous = "==1.1.0"
+urllib3 = ">=1.24.2"
```

Comment:
Out of curiosity, did `pipenv update` not update this dependency?

Reply:
@andrewoh531 should be better now.